src/components/content/common/ocl/DisplayOclData.tsx

```diff
@@ -141,6 +142,11 @@ function DisplayOclData({ ocl }: { ocl: Ocl }): React.JSX.Element | string {
                             </Descriptions.Item>
                         </Descriptions>
                         <DeploymentVariables variables={ocl.deployment.variables} />
+                        {ocl.serviceConfigurationManage?.configurationParameters ? (
+                            <DeploymentServiceConfigurationParameter
+                                parameters={ocl.serviceConfigurationManage.configurationParameters}
+                            />
+                        ) : null}
                         <FlavorsText flavors={ocl.flavors.serviceFlavors} />
```

Comment:
i think it is better we move the flavors above Deployment variables and configuration parameters. otherwise the flavors are kind of lost.